### ISSUE TITLE
Support for SVGs

### ISSUE THREAD

**Issue by kmnaseef** [saket/telephoto#31]
Adding below lines, .gif and .svg files can be opened with coil AsyncImage.

```
imageLoader = ImageLoader.Builder(context)
    .components {
        add(SvgDecoder.Factory())  // svg
        add(ImageDecoderDecoder.Factory()) // gif
    }
    .build()
```
but with  ZoomableAsyncImage,  gif files can be played without any problem, but svg files not open.

**Comment by saket** [comment]
I haven't investigated this yet, but I think my decision to enable sub-sampling for all bitmaps may be causing this. Let me fix this.

**Comment by kmnaseef** [comment]
Ok. Thanks...

**Comment by saket** [comment]
Fixed in https://github.com/saket/telephoto/commit/e1ad337aa4edce4f1a33f6aaee42af7305c546bd.

**Comment by saket** [comment]
Thanks for reporting this @kmnaseef! I'll release the fix as part of `0.5.0` soon. In the meantime, you can unblock yourself by using `0.5.0-SNAPSHOT`.

**Comment by kmnaseef** [comment]
You're welcome bro..
